### PR TITLE
Update Docs: Reference “configuration-options”

### PR DESCRIPTION
**Pull request by joelmukuthu**
Automatically generated by Netlify CMS

### CODE REVIEW

File: website/content/docs/configuration-options.md [hunk 1]
```diff
@@ -18,7 +18,7 @@ You can find details about all configuration options below. Note that [YAML synt
 
 ## Backend
 
-_This setting is required._
+*This setting is required.*
```
Comment (joelmukuthu) [inline website/content/docs/configuration-options.md]: I didn't make these changes (the only change I made was on line 33), there might be a bug that caused this.

What I did:

1. Clicked the "Edit this page" button on this page https://www.netlifycms.org/docs/configuration-options/
2. Signed in with GitHub
3. Gave authorisation for netlify-cms to access my public repos on GitHub
4. Clicked on "Fork the repo"
5. Made the changes (on line 33)
6. Saved my changes
7. Set the status to "In review"

Should I report this as a bug? Did I do something wrong somewhere?

Comment (erezrokah) [reply website/content/docs/configuration-options.md]: That's actually the way the CMS serializes markdown and has no difference:
https://www.markdownguide.org/basic-syntax/#italic.

Comment (erezrokah) [reply website/content/docs/configuration-options.md]: You can see the preview for your change here https://deploy-preview-3317--netlify-cms-www.netlify.com/docs/configuration-options/#publish-mode

(Available by clicking `details` on the `netlify/netlify-cms-www/deploy-preview` PR status)

Comment (joelmukuthu) [reply website/content/docs/configuration-options.md]: Well. I saw the `-` at the beginning of the removed line and thought the serialisation was getting rid of the markdown lists :laughing: 

Thank you for the info though! :raised_hands:

**Comment by erezrokah** [comment]
Thanks @joelmukuthu!

**Pull request merged**